docs/marketplace/allowlist_what_and_why.rst

```diff
@@ -0,0 +1,27 @@
+============================================
+Allowlist: what's it and why is it needed?
```

Comment:
```suggestion
Allowlist: what is it and why is it needed?
```
Guessing this was a Vale correction but actually makes better grammatical sense to not abbreviate in that way.